### PR TITLE
Adds parentheses in expressions inside template literals

### LINKED ISSUE #28: Does paren space work correctly inside tagged templates?
Via https://github.com/Automattic/wp-calypso/pull/42694#pullrequestreview-419011075

Looking at the prettier playground, it does appear to format these so perhaps inserting space in the `${}` is something `wp-prettier` is missing. [The prettier playground formats these lines identically](https://prettier.io/playground/#N4Igxg9gdgLgprEAuc0DOMAEBlADgQzDgHkAnAOQj0LkwF5MMBPAGzgBMA6dgSwDcABgB0oQmDFL4oaAGYRSAWySYAJMAAUuUhFxoAlHQB8m7bs480AYQgLcbeAH4A5Npj54ATXUBGABwAGdjgAcz0nJBcIN091fzC9AF8AbhFMNMwJKVl5JVVgTHVMLR00TD16QwKi0zRzKxs7OHhMB0xI6LgvP0CQsMxldvdO2L7y5JEBFKgQABoQHRgedGRQfFJtAHcABTWENGQQfBYN-CZ9uYAjSTAAaybqMB4oYOQJAFc4OYALGAUWAHUvjx4GgCERsHtgfxgUwDmA0OcQE80HBSDAtpJggp8MgZEcUXMAFZoAAeACFrncYNh8Ao4AAZJ5wXH4z4gYkk7BPYJsACKbyizKQeJYBJABFIKNIBwu+AucBYs3FpCeMH+PHYMC+yACc2KKP+klwBy0cClfGZcwAjgL4BiSgd8GgALRQOAcDhK0hwG08b0Y-BYnHC1lzFEKHivUgfMPcvm2oUisVuC7qzXapAAJjmmR4LG51gU2IOZoArEq3iiACpy-Yh0VsvgfACSUCCsGwYBVuBgAEE29gYKxE6yEgkgA), as I'd expect:

```js
const SpaceOrNoSpace = styled.div`
  transform: ${(props)=>(props.isComplete?'rotateY(180deg)':'rotateY(0)')};
  transform: ${ ( props ) => ( props.isComplete ? 'rotateY(180deg)' : 'rotateY(0)' ) };
`;
```

### PR DESCRIPTION
**Pull request by scinos**
Fixes #28 

Allows using spaces after parentheses in expressions inside template literals.

### CODE REVIEW

**Review by sirreal — COMMENTED** [review]
Thanks! Tests seem to indicate this is working as expected, but I'd love to have @jsnajdr review.

**Comment by sirreal** [comment]
I've checked, added tests for, and pulled in a few more changes from 01fd780c7c5a8edd9e3ef7daea49e650fa42cee5. Formatting changes courtesy of @jsnajdr 🙇‍♂️

**Review by sirreal — APPROVED** [review]
After comparing to prior work in 01fd780c7c5a8edd9e3ef7daea49e650fa42cee5 and checking the test results, this looks good to me. Thanks @scinos and @jsnajdr 🙌

File: tests/paren_space/jest-test-each.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+test.each`
```
Comment (scinos) [inline tests/paren_space/jest-test-each.js:1]: TIL

Comment (jsnajdr) [reply tests/paren_space/jest-test-each.js:1]: Oh yes, `test.each` is a weird feature of Jest 🙂

**Pull request merged**